css/templates/land-see/page.css

```diff
@@ -354,3 +354,115 @@ amp-img.cover img {
 .ampstart-input input[type="email"].user-invalid {
   border: 2px solid var(--color-form-invalid);
 }
+
+/* Accordion */
+
+.accordion section {
+  margin-bottom: 10px;
+  position: relative;
+  overflow: hidden;
+}
+
+.accordion section > h3 {
+  text-transform: uppercase;
+  font-weight: bold;
+  min-height: 68px;
+  line-height: 68px;
+  outline: none;
+}
+
+.accordion section > h3:after {
+  content: '';
+  position: absolute;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M6.967 13.932V0M0 6.966h13.932'/%3E%3C/svg%3E");
+  height: 14px;
+  width: 14px;
+  right: 30px;
+  top: 28px;
+  cursor: pointer;
+}
+
+.accordion section[expanded] > h3:after {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M0 6.966h13.932'/%3E%3C/svg%3E");
+}
+
+.accordion section:before {
+  content: '';
+  position: absolute;
+  height: 168px;
+  left:0;
+  right: 0;
+  background-color: var(--color-accordion-color-base);
+}
+
+.i-amphtml-accordion-header { /* reset defaults */
+  cursor: default;
+  background-color: inherit;
+  padding-right: 0;
+  border: none;
+}
+
+section.accordion-bg-100:before {
```

Comment:
again `accordion-bg-100` is not a good class name - ideally namespace your classes and name based on functionality not utility  (the only utility classes we allow are from BASS CSS.)